run_interactive.py

```diff
@@ -75,6 +75,9 @@
 _SHARDING_CONFIG = flags.DEFINE_string(
     "sharding_config", "", "config file for sharding"
 )
+_SHARD_ON_BATCH = flags.DEFINE_bool(
```

Comment:
Nit, can you add `when enabled, it overwrites the sharding_config` or something similar?

Reply:
done